packages/dynamic_layouts/test/dynamic_grid_test.dart

```diff
@@ -134,7 +134,8 @@ class TestSimpleLayout extends DynamicSliverGridLayout {
   bool reachedTargetScrollOffset(double targetOffset) => true;
 
   @override
-  SliverGridGeometry updateGeometryForChildIndex(int index, Size childSize) {
+  DynamicSliverGridGeometry updateGeometryForChildIndex(
+      int index, Size childSize) {
```

Comment:
nit: add trailing comma after `childSize` to get better formatting from the formatter.